jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-api/src/main/java/org/jbpm/console/ng/cm/util/CaseActionSearchRequest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.console.ng.cm.util;
+
+import org.jboss.errai.common.client.api.annotations.Portable;
+import org.jboss.errai.databinding.client.api.Bindable;
+
+@Portable
+@Bindable
+public class CaseActionSearchRequest {
+
+    private Integer status = 1;
+
+    private CaseActionsFilterBy filterBy = CaseActionsFilterBy.AVAILABLE;
+
+    private String sort = "createdOn";
+
+    private boolean sortOrder = true;
```

Comment:
Maybe renaming this filed to "sortAsc" (as you named the local variable in getCaseActionsByStatusTest() which contains the same value) would increase readability. 
"sortOrder" being set to "true" implies more that sorting is enabled in general and not that it just means ascending order.